prettier/prettier-check.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: prettier-check-from-gitrepo
+spec:
```

Comment:
I suggest adding a `description` field under the `spec` that describes in plain english what this Task (and prettier) does.  Something like the following:

```
description: Run the prettier code formatter against provided source code. See https://prettier.io for more information.
```